src/cobalt_model/pagination_config.rs

```diff
@@ -0,0 +1,155 @@
+use std::convert::Into;
+use std::vec::Vec;
+
+use super::SortOrder;
+
+pub const DEFAULT_PERMALINK: &str = "{{parent}}/{{include}}/_p/{{num}}/";
+pub const DEFAULT_SORT: &str = "published_date";
+pub const DEFAULT_PER_PAGE: i32 = 10;
+
+#[derive(Copy, Clone, Debug, Serialize, Deserialize, PartialEq, Eq)]
+#[serde(deny_unknown_fields)]
+pub enum Include {
+    None,
+    #[serde(rename = "all")]
```

Comment:
Do we nornally do `rename` for our enums?  Whatever we specified, I think I'd rather we just be consistent.